api/init_sql/scout.sql

```diff
@@ -72,6 +72,7 @@ CREATE TABLE `reports` (
   `conditions_count` int(11) DEFAULT NULL,
   `type` varchar(128) NOT NULL DEFAULT '',
   `fields_to_select` longtext,
+  `show_in_dashboard` boolean NOT NULL DEFAULT 0,
```

Comment:
Not a big deal, but I think the rest the bool fields use `tinyint(1)` - should end up the same though